### PR TITLE
[AI-5] FDM v3 support

### PR DESCRIPTION
**Pull request by dlcognite**
https://cognitedata.atlassian.net/browse/AI-5

**Problem / User story**
Data modeling (FDM GA) is coming, and Granafa should support what it did on top of DMS v2 and Templates

**Solution**
Refactored DMS v2 glow to fit DMS v3 (data modeling in CDF)

**Affected Resource types**
FDM

**Changes to datasource JSON model**
FDM, breaking change to v2 (removed in favor for the GA flow)

**Has documentation been updated**
NO, please help and educate me how 🙏 

**Have tests been updated?**
yes

**Screenshots and examples**
Screenshots and example query if applicable

image

Using `platypus` on greenfield (ask for full credential from @dlcognite ), use the above Data Model and version, with the following query
data model:  `Sample_Data_Model_With_Types` in space `Sample`
version: 3
query:
```graphql
query MyQuery {
  listWithPrimitives {
    items {
      timeseries {
        __typename
        externalId
      }
    }
  }
}
```

### CODE REVIEW

**Comment by dlcognite** [comment]
No worries yet! We are waiting for a few features in the backend still, will ping when I need review!

**Comment by dlcognite** [comment]
working with timeseries 
image

**Comment by dlcognite** [comment]
@polomani good sir, how do i document plugins and update screenshots?

**Comment by dlcognite** [comment]
also fixed the header style, was giving me anxiety 😓 
image

**Comment by polomani** [comment]
@dlcognite the documentation is moved here https://docs.cognite.com/cdf/dashboards/guides/grafana/getting_started, so we don't necessarily have to merge the at the same time, but at least have some draft maybe?
Considering that this is GA soon, it's should be not a bad idea to document it officially?

**Comment by polomani** [comment]
@dlcognite regarding screenshots, here are the specs:
https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/#screenshot-guidelines-1

Just put them in the images folder, I can help with referencing it in the plugin.json afterwards (or just look here https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#properties-5)

**Comment by muradsater** [comment]
Should we remove "Preview" from Data Models and enable by default?  It will cause lots of questions if it's hidden behind the toggle in data source settings.

File: README.md [hunk 1]
```diff
@@ -43,7 +42,7 @@ ln -s /path/to/cognite-grafana-datasource cognitedata-datasource
 
 `yarn` followed by `yarn build` should work on systems with a shell.
 
-For debugging and development, use `yarn dev:watch`, and for testing use `yarn test`.
+For debugging and development, use `yarn watch`, and for testing use `yarn test`.
```
Comment (polomani) [inline README.md:45]:
```suggestion
For debugging and development, use `yarn dev`, and for testing use `yarn test`.
```

File: src/components/configEditor.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ const enableTemplatesTooltip = `Enable the templates tab for use with the Cognit
 
 const enableEventsAdvancedFilteringTooltip = `Enable the Events advanced filtering (preview)`;
 
-const enableFlexibleDataModellingTooltip = 'Enable Flexible Data Modelling (preview)';
+const enableFlexibleDataModellingTooltip = 'Enable Data Models (preview)';
```
Comment (polomani) [inline src/components/configEditor.tsx:37]:
```suggestion
const enableFlexibleDataModellingTooltip = 'Enable Data Models';
```

**Review by polomani — APPROVED** [review]
Nice work!

**Pull request merged**